repo-scripts/size-analysis/bundle-definitions/firestore.json

```diff
@@ -98,5 +127,108 @@
         ]
       }
     ]
+  },
+  {
+    "name": "Query Cursors",
+    "dependencies": [
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "app",
+            "imports": [
+              "initializeApp"
+            ]
+          }
+        ]
+      },
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "firestore",
+            "imports": [
+              "getFirestore",
+              "collection",
+              "doc",
+              "getDoc",
+              "query",
+              "orderBy",
+              "startAt",
+              "endBefore"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "name": "Persistence",
+    "dependencies": [
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "app",
+            "imports": [
+              "initializeApp"
+            ]
+          }
+        ]
+      },
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "firestore",
+            "imports": [
+              "getFirestore",
+              "enableIndexedDbPersistence",
+              "collection",
+              "onSnapshot",
+              "where",
+              "query",
+              "enableNetwork",
+              "disableNetwork"
```

Comment:
I think we should remove all those - they are tested elsewhere.

Reply:
Done.